### ISSUE TITLE
qǝne poems

### ISSUE THREAD

**Issue by thea-m** [BetaMasaheft/Documentation#2504]
I am describing a manuscript (Halle, DMG Yi 135 III C (7)) that consists of c. 20 qǝne poems, varying in length between a few lines and one page.
Should I create work records for each (on the grounds that the text is available), or rather use narrative units for the indicated qǝne modes (ʿǝṭāna mogar, gubāʿe qānā etc) and give the text of the poems only in the manuscript description (on the grounds that they presumably were not composed/written down with an intention of further circulation)?

See also https://github.com/BetaMasaheft/Documentation/issues/1289

@DenisNosnitsin1970 @CarstenHoffmannMarburg @eu-genia

**Comment by eu-genia** [comment]
Thank you!

Qǝne is always a tricky matter, being an oral composition put in writing...

On going through the records I see that there are indeed many inconsistencies that have to be cleaned. 

1. Many of the manuscripts with qǝne poems were described using keywords/authority files. Yet there is a keyword `PoemQene` https://betamasaheft.eu/authority-files/PoemQene/main, (this one is the old one, with a description to it), and then probably following #1289 an additional keyword `Qene `https://betamasaheft.eu/authority-files/Qene/main was created, both have been used in the same set of manuscripts, with one exception, see
https://betamasaheft.eu/authority-files/list?keyword=PoemQene and https://betamasaheft.eu/authority-files/list?keyword=Qene (the second list has one more, BLorient570) - neither of the lists is complete, there are many MSS I know have qǝne poems but do not feature on either list, so probably the keywords have not been used consistently.

2. There is also a general CAe ID `LIT5129Qene ` https://betamasaheft.eu/works/LIT5129Qene/main used so far in 2 manuscripts.

3. Then there is also still `LIT4013EtanaM `and possibly other similar work records (Issue #1289 was opened and closed but we still have `LIT4013EtanaM `in CAe, used also in manuscript records, despite it being a type of text and not a text, `t2 `and `t3 `being however incipits of texts never transformed to either LIT or NAR, somewhat confusing)

4. Most manuscripts mention qǝne use neither keyword nor ID of any type, e.g. see `BNFabb145 `with a collection of qǝne  produced for d'Abbadie and edited by Guidi, no ID in Bm


My suggestions here are:
1. remove one of the keywords, stick to one
2. use the remaining keyword (possibly `PoemQene`, having a more elaborate structure, or `Qene`, but then the abstract should be copied over) consistently in all cases where a qǝne poem is featured, to be able to easily look for them (so all these currently 59 mss https://betamasaheft.eu/as.html?query=qene&homophones=true&AttestedInType=1&xmlid=&work-types=mss should appear in the keyword search results too)
3. create and use upper-level general CAe IDs for poems that cannot be further identified (`LIT5129Qene `for any unspecified poem, `LIT4013EtanaM `for any unspecified qǝne  of etana mogar type, etc. for each type)
4. create IDs for recorded poems that can be clearly identified, even if these are unique, as we also do for letters and other unique texts, as long as they are part of the main content of a manuscript (I would not be so sure about `addition`es though, we usually do not create upper level IDs for prayers/greetings/supplications appearing uniquely in additiones if they are not copied more times and/or as main text - so for `addition`es I would leave the keyword and the text, where available, as a `q`)
4.1. one way to do this is to create upper level CAe ID for each poem (am not sure whether this is the best way)
4.2. the more economic solution could be that we add an `edition` in the generic records and there list the poems we encounter, each with its own `div`,  and naturally `xml:id`, and then we can refer to them elsewhere with that derived ID (e.g. looking at the examples given here https://betamasaheft.eu/authority-files/PoemQene/main we take the text from the example one, put it as a `div `  with `xml:id="ESmma008"` under `LIT4013EtanaM `and then we can refer to it as `LIT4013EtanaM#ESmma008` in the `msDesc`

**Comment by DenisNosnitsin1970** [comment]
Of course, we should remember that in most of the cases the type of qene remains not identified. In Abb 145 it is not; Guidi who edited them did not identified the types either; those with type- identification are rather exceptions. To identify the type is a difficult and special task. This was one of the previous considerations, as far as i can remember.

**Comment by DenisNosnitsin1970** [comment]
Indeed, there can be the means to sort out those whose type are known, according to the types but those who are not can be made just good searchable. If qenes are part of a work (= main content), this is again a different matter.

**Comment by eu-genia** [comment]
@abausi should also have a say as in fundamental decisions that regard (or not) the Clavis

**Comment by DenisNosnitsin1970** [comment]
I volunteer to follow the topic closer as I have to prepare smth about it.

**Comment by abausi** [comment]
Ok, Denis; as for me, I would agree with 1), 2), 3) and 4) as suggested by Genia, If it is a secondary text, that weill be noted, but (like for a short miracle), it has to receive an ID as well. Important also to have a definition of the type, obviously, if this is enough clear, and for all types, of course. I am only undertain whether also 4.2 really works, but it might be a good proposal.

**Comment by thea-m** [comment]
Thank you.
If there is agreement that @DenisNosnitsin1970 will elaborate the general encoding, let me just come back to those suggestions that pertain directly to the practical current issue of the manuscript in question. 
In a choice between 4-1 and 4-2, I would prefer 4-1, because 4-2, while elegant encoding, would be very idiosyncratic within our research environment, and I think there is an argument to be made for solutions also to be understandable to those who are not encoding experts.
Regarding the manuscript DMG Yi 135 III C (7), this would mean to create single CAe IDs for all poems contained in it (all are identified with a qǝne type in the manuscript), which I will do if there are no objections.

**Comment by eu-genia** [comment]
(the images in EMIP01908 should be visible now, as mentioned, we have all EMIP images but many stubs lack the information on the number in `facs`, as soon as this is added the images become accessible, check https://github.com/BetaMasaheft/Documentation/issues/2017 )

to sum up
-We always must use the keyword (`Qene `or `PoemQene`?) - this will make the already searchable if one wants to find all qene we know of
-When part of the main text, a CAe ID must be used. 
--When we have no access to the text we can use LIT5129Qene or, when we know the mode, a generic ID for the mode (?)
--When we have the text (which I meant under "clearly identified" we should create a specific CAe ID).
-When an additio, type `PoemQene ` with the `q` wherever possible is sufficient

**Comment by DenisNosnitsin1970** [comment]
Having surveyed the materials a little, I still have a couple of questions to suggestions, to be asked after the break.

**Comment by DenisNosnitsin1970** [comment]
Just to continue and get clear guidelines: we assign IDs to all qenes, shaped as additional notes ("GuestText") or a part of main text; the same with the collections (ideally each with an ID, alternatively as text parts). There should be a possibility to indicate, in a formalized way, the mode of qene, if known,  inside the record. The simplest way is to use LIT5129Qene as general record for the cases where the text proper of a qene is not available, or, unless considered too odd, also in this case an ID can be created (esp. if the mode is known).

**Comment by thea-m** [comment]
Maybe you could create keywords for the modes, in analogy to the keywords ArarayMode etc?

**Comment by DenisNosnitsin1970** [comment]
I see that in the current work records for qene the poem is represented as continuous text, without line structure (despite having been encoded according to the line structure (`<l n="1"/>` etc.). Can that be changed? Keywords for the qene types would be great.

**Comment by eu-genia** [comment]
The schema and the taxonomy now have the keywords

```
<category>
         <desc>Qǝne Types</desc>
         <category xml:id="EtanaMogar" corresp="https://betamasaheft.eu/authority-files/EtanaMogar/main">
            <catDesc>ʿƎṭāna mogar</catDesc>
         </category>
         <category xml:id="Mawaddes" corresp="https://betamasaheft.eu/authority-files/Mawaddes/main">
            <catDesc>Mawaddǝs</catDesc>
         </category>        
         <category xml:id="KebrYeeti" corresp="https://betamasaheft.eu/authority-files/KebrYeeti/main">
            <catDesc>Kǝbr yǝʾǝti</catDesc>
         </category>
         <category xml:id="Sellase" corresp="https://betamasaheft.eu/authority-files/Sellase/main">
            <catDesc>Śǝllase</catDesc>
         </category>
         <category xml:id="Zayeeze" corresp="https://betamasaheft.eu/authority-files/Zayeeze/main">
            <catDesc>Za-yǝʾǝze</catDesc>
         </category>
      </category>
```

I have gone through some of the manuscripts to insert these, we need to check works and narratives, e.g. LIT4013EtanaM, NAR0183EtanaMogar, see also https://github.com/BetaMasaheft/Works/pull/1086, to disambiguate and clean

**Comment by eu-genia** [comment]
In general, of course when quoting poetry it is better to use mark up for the lines (at least as `<lb/>`)

**Comment by eu-genia** [comment]
- In the end `LIT5129Qene `was used for a specific qǝne, and `LIT4013EtanaM `will also become a specific case, so we have no general IDs - if necessity arises these may be created; for additiones linking to the keyword is sufficient
- Keywords: we have `PoemQene `as part of the taxonomy of additions and `Qene` (and now additionally also the types) in the genres taxonomy. It is probably not helpful to confuse the taxonomy, so I have disambiguated the description in the Authority Files to make the difference clear. Ideally for additiones it should be something like `<desc type="PoemQene"> A <term key="Qene">qǝne</term> of <term key="Sellase"> ሥላሴ፡</term> type </desc>`

**Comment by eu-genia** [comment]
refresh all your repositories and click 
![image](https://github.com/BetaMasaheft/Documentation/assets/17987728/62886915-c683-4a27-b130-0f70629ce193)

**Comment by eu-genia** [comment]
`Kwellekomu`

**Comment by eu-genia** [comment]
sorry there is an o

**Comment by eu-genia** [comment]
https://github.com/BetaMasaheft/Works/pull/1123#issuecomment-2181592568

**Comment by DenisNosnitsin1970** [comment]
NAR0183EtanaMogar is not used at all. According to my understanding of what is "narrative unit", I see that there is a very, very narrow chance that it will ever come to use. Should it be left or removed, to get it less chaotic?

**Comment by eu-genia** [comment]
See above: we said we use LIT4013EtanaM wherever we have no text for identification, and create independent CAe IDs when we have.
NAR0183EtanaMogar I think yes it can be deleted.

**Comment by DenisNosnitsin1970** [comment]
That is ok, can/should the way we use the record be somehow reflected in it? that this is a general record for the qene type, to be used when we have no access to the qene text. (I can do this).  Otherwise it may be confusing.

**Comment by eu-genia** [comment]
OK. "general record for the qene type, to be used when we have no access to the qene text"

**Comment by eu-genia** [comment]
https://github.com/BetaMasaheft/Documentation/issues/2614#issue-2493537950 Gubae Qana

**Comment by abausi** [comment]
I am in favour of creating the entry.

Il ven 22 mar 2024, 14:15 Eugenia ***@***.***> ha scritto:

**Comment by CarstenHoffmannMarburg** [comment]
We probably need a keyword for Mi-bazḫu-type of Qəne. Cf. LIT7225EklaAbasaha in https://github.com/BetaMasaheft/Works/pull/1371/files

**Comment by eu-genia** [comment]
If you see the commit two lines above your comment - it has been already done. https://github.com/BetaMasaheft/Authority-Files/commit/a40d1d066394d27969666ed1bd36975d9aeb31b1 

Please check the discussion before updating it, thank you!

**Comment by CarstenHoffmannMarburg** [comment]
I think, we need another authFile for the Qene-type Za-ʾAmlākiya, because thanks to Denis, we found a specimen in EMIP00499, f. 160r which has three verses and is indicated with this type

See definition in Guidi: 

![Image](https://github.com/user-attachments/assets/5874c0c2-1067-4110-8bda-814744f9290a)

And text in EMIP00499:

![Image](https://github.com/user-attachments/assets/6f892173-ca98-4fb4-8345-59fa276d596d)

Cf. our discussion in https://github.com/BetaMasaheft/Documentation/issues/2830.

**Comment by CarstenHoffmannMarburg** [comment]
In case Śāhlǝka is truely an indication of a qǝne-type (with three verses), we may also need a new authFile Sahleka.

![Image](https://github.com/user-attachments/assets/fc31cdf2-d390-43d8-82b3-4857d46cdcdf)

Cf. https://github.com/BetaMasaheft/Works/pull/1509. N.B. not listed in Guidi 1901.

**Comment by CarstenHoffmannMarburg** [comment]
> N.B. not listed in Guidi 1901.

As Denis told me, śāhlǝka is indeed mentioned in Guidi 1901 on page 4, but the information is somehow confusing to me. It appears, that he saw it as a subgroup of the mawaddǝs-type without specific information on the number of verses:

![Image](https://github.com/user-attachments/assets/0398b339-e611-44e5-88ed-fee52b3d5f2c)

In an Amharic book about Qǝne by Denis, śāhlǝka is mentioned with three verses.

Can I create record for Za-ʾAmlākiya and one record for śāhlǝka or does someone else take this task?

**Comment by DenisNosnitsin1970** [comment]
Yes, the information is a bit confusing (actually not so much, because for Guidi this is one of the mawaddes subtypes, but for our system it does not change anything), and there is no info on the number of verses, but I sent to you an excerpt from an Ethiopian book where it is listed. The trick is that za-amlakiya has three verses and sahleka has also three (and even more). I do not know how to distinguish them (probably according to the function), but we just follow indications in the mss, as above. Maybe the author defined it as "sahleka" and not other type with three verses exactly because it was supposed to be linked with a Deggwa chant (the exact situation known only to the author).

**Comment by DenisNosnitsin1970** [comment]
You can create those two records, sure! Thank you.

**Comment by CarstenHoffmannMarburg** [comment]
> but I sent to you an excerpt from an Ethiopian book where it is listed.

Do you speak about bm:Afawarq1984Yageezqeneyat? Can you give me the page numbers as well?
